app/controllers/api/v1/ratings_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Api::V1::RatingsController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
+
+  before_action :authenticate_user, only: [:update]
+
+  def create
+    location_to_update = Location.find(params[:location_id])
+    new_rating = params[:_json]
+
+    submit_rating = Rating.new(
+      user: current_user,
+      location: location_to_update,
+      rating: new_rating)
+
+    record = Rating.where(
+      user_id: current_user.id,
+      location_id: location_to_update.id,
+      rating: new_rating).exists?
+
+    if record == true
```

Comment:
Doesn't require a fix, but fyi that this line is exactly equivalent to `if record` (the `== true` is built into the conditional)